src/component/index.js

```diff
@@ -86,4 +93,11 @@ Toggle.propTypes = {
   id: PropTypes.string,
   'aria-labelledby': PropTypes.string,
   'aria-label': PropTypes.string,
+  icons: PropTypes.oneOfType([
+    PropTypes.bool,
```

Comment:
I think this should be `PropTypes.oneOf([false])` instead, since we don't expect `true` to be passed in. Not sure if there's a cleaner way to write that, but couldn't find anything in [the docs](https://facebook.github.io/react/docs/reusable-components.html).